src/main/java/net/minecraftforge/common/ForgeConfig.java

```diff
@@ -108,10 +108,32 @@
                     .comment("Fix advancement loading to use a proper topological sort. This may have visibility side-effects and can thus be turned off if needed for data-pack compatibility.")
                     .translation("forge.configgui.fixAdvancementLoading")
                     .define("fixAdvancementLoading", true);
+
             builder.pop();
         }
     }
 
+    /**
+     * General configuration that doesn't need to be synchronized but needs to be available before server startup
+     */
+    public static class Common {
+        public final ForgeConfigSpec.ConfigValue<? extends String> defaultWorldType;
+
+        Common(ForgeConfigSpec.Builder builder)
+        {
+            builder.comment("General configuration settings")
+                    .push("general");
+
+            defaultWorldType = builder
+                    .comment("Defines a default world type to use. The vanilla default world type is represented by 'default'.")
```

Comment:
Would it be worth noting in the comment that modded world-types must be specified with a namespace? Modders will need to relay that information anyway (for server.properties) but there could still be confusion since "default" isn't prefixed with "minecraft:"

Reply:
Good idea. It may be confusing otherwise for those who are used to writing "biomesoplenty", which would mean "minecraft:biomesoplenty" with this change.